main.js

```diff
@@ -1,34 +0,0 @@
-//variable declarations
-let contButtons = Array.from(document.getElementsByClassName('continue-reading-button'))
-let learnButtons = Array.from(document.getElementsByClassName('learn-more-button'))
-let singlePostButtons = Array.from(document.getElementsByClassName('date-button'))
-let notHomeHeaders = Array.from(document.getElementsByClassName('not-home'))
-let tellMeMore = Array.from(document.getElementsByClassName('tell-me-more-btn button'));
-
```

Comment:
I am extremely disappointed in the both of you that you didn't embed Rick Astley's famous "Never Gonna Give You Up" on this site to Rick Roll anyone who visits it.